src/components/Dashboard/Post/Post.jsx

```diff
@@ -22,90 +21,76 @@ import ErrorIcon from "@material-ui/icons/Error";
 import CheckCircleIcon from "@material-ui/icons/CheckCircle";
 import { useStyles } from "./Style";
 
-export default function RecipeReviewCard() {
+export default function RecipeReviewCard({ post }) {
   const classes = useStyles();
-  const tileData = [
-    {
-      id: 1,
-      img:
-        "https://cosmosmagazine.com/wp-content/uploads/2020/02/190404-cat-full.jpg",
-    },
-    {
-      id: 2,
-      img:
-        "https://cosmosmagazine.com/wp-content/uploads/2020/02/190404-cat-full.jpg",
-    },
-    {
-      id: 3,
-      img:
-        "https://cosmosmagazine.com/wp-content/uploads/2020/02/190404-cat-full.jpg",
-    },
-  ];
-  const isLost = true;
   return (
     <Card className={classes.root}>
-      <CardHeader
-        avatar={<Avatar src={ghost} className={classes.avatar} />}
-        action={
-          <IconButton aria-label="settings">
-            {/* <MoreVertIcon /> */}
-            <MdMoreVert />
-          </IconButton>
-        }
-        title="Othman"
-        subheader={
-          <>
-            <div>Sep 14 2021, Erbil</div>
-            {isLost ? (
-              <Chip
-                className={classes.lostTag}
-                color="secondary"
-                size="small"
-                icon={<ErrorIcon className={classes.tagIcon} />}
-                label="Lost"
-              />
-            ) : (
-              <Chip
-                className={classes.foundTag}
-                color="primary"
-                size="small"
-                icon={<CheckCircleIcon className={classes.tagIcon} />}
-                label="Found"
-              />
-            )}
-          </>
-        }
-      />
-      <CardContent className={classes.content}>
-        <Typography variant="body2" color="textSecondary" component="p">
-          This impressive paella is a perfect party dish and a fun meal to cook
-          together with your guests. Add 1 cup of frozen peas along with the
-          mussels, if you like.
-        </Typography>
-      </CardContent>
-      {/* <CardMedia
+      <div>
+        <CardHeader
+          avatar={<Avatar src={post.avatar} className={classes.avatar} />}
+          action={
+            <IconButton aria-label="settings">
+              {/* <MoreVertIcon /> */}
+              <MdMoreVert />
+            </IconButton>
+          }
+          title={post.name}
+          subheader={
+            <React.Fragment>
```

Comment:
Use <> instead of React.Fragment!

Reply:
Fixed 7c3841e95070633f0f9b2f8da4bb63e578653590